core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -69,10 +69,10 @@
     <div ng-cloak class="navbar-header float-right">
       <ul class="nav oppia-navbar-nav oppia-navbar-profile-admin">
         <li>
-          <ul class="nav oppia-navbar-tabs-admin oppia-admin-page-dropdown" uib-dropdown>
+          <ul class="nav oppia-navbar-tabs-admin oppia-admin-page-dropdown" uib-dropdown ng-class="{'open':ctrl.dropdownMenuisActive}">
             <li uib-dropdown-toggle class="nav-item oppia-clickable-navbar-element"
-                ng-mouseover="$ctrl.onMouseoverDropdownIconOrMenu($event)"
-                ng-mouseleave="$ctrl.onMouseoutDropdownIconOrMenu($event)">
+                ng-mouseenter="ctrl.dropdownMenuisActive = true"
```

Comment:
Have you tested these changes? I was expecting it to be $ctrl*

Reply:
My bad, I have tested changes, and they are working the same as before(before $ctrl), updating ctrl -> $ctrl